lib/kitchen/element_base.rb

```diff
@@ -92,6 +92,14 @@ class ElementBase
     #   @return [Selectors::Base]
     def_delegators :config, :selectors
 
+    # @!method pantry
+    #   Access the pantry for this element's document
+    #   @return [Pantry]
+    # @!method :clipboard
+    #   Access the clipboard for this element's document
+    #   @return [Clipboard]
+    def_delegators :document, :pantry, :clipboard
```

Comment:
@jpslav I added a `:clipboard` but I couldn't actually update any calls to `document.clipboard` because on line 682 `clipboard` is a protected method.
Should I remove `:clipboard` from line 101 then or do you have another suggestion?